src/components/CopyButton.vue

```diff
@@ -0,0 +1,76 @@
+<template>
```

Comment:
for now, using pug messes too much with my spatial sense of how markup is structured haha

Reply:
Fine for now, we can always cleanup later.